src/ucs/type/class.c

```diff
@@ -54,7 +54,7 @@ void ucs_class_call_cleanup_chain(ucs_class_t *cls, void *obj, int limit)
 
 void *ucs_class_malloc(ucs_class_t *cls)
 {
-    return ucs_malloc(cls->size, cls->name);
+    return ucs_memalign(0x32, cls->size, cls->name);
```

Comment:
0x32 != 32..
also interesting why clang thinks it's aligned, maybe we confused it with __attribute__((align)) in UD code

Reply:
you are reviewing too fast :)
I wanted just copy changes to docker system

uct_ud_verbs_iface_t is not aligned by attribute...

Reply:
> 0x32 != 32..

This is good one :-)

Reply:
yep, Yossi catched changes when I was experimenting with alignments :)